qulice-pmd/src/main/resources/com/qulice/pmd/ruleset.xml

```diff
@@ -256,4 +256,26 @@ OF THE POSSIBILITY OF SUCH DAMAGE.
       </property>
     </properties>
   </rule>
+  <rule name="ConstantsOnRightSideOfComparison" language="java" class="net.sourceforge.pmd.lang.rule.XPathRule" message="Constants should be on the right side of comparisons.">
+    <description>
+      Enforces the code style guideline that constants (null, string literals, or numbers) should appear on the right side of comparison operators to reduce the risk of accidental assignment and improve readability.
```

Comment:
- It is better to have this rule only about numbers and null values (because for string values there is another rule, that will write that you need to use equals method instead of ==) (So if you have code `` "some" == some``, you will get the first warning to revert it, and the second - use equals and it can be confusing)
- The same rule should be about != operation, as they are similar, so ``null != some`` should produce the same error